lib/pharos/transport/base.rb

```diff
@@ -20,12 +20,10 @@ class Base
 
       attr_reader :host
 
-      # @param host [String]
-      # @param opts [Hash]
-      def initialize(host, **opts)
+      # @param host [Pharos::Configuration::Host]
+      def initialize(host)
```

Comment:
Not related to autoload.

Reply:
We can bring back transport.rb and remove the requires.